### ISSUE TITLE
missing or invalid headings for top-level pages

### ISSUE THREAD

**Issue by thatbudakguy** [Princeton-CDH/mep-django#640]
from [john's review](https://docs.google.com/spreadsheets/d/1N-oBy9Mp91TEdXEi6rjCMNsbJNacxMh6CEGTHpCRfws/edit?usp=sharing):
> Some of the top level pages have no or little heading structure. Wrap the site name in an `<h1>` on the homepage. Add hidden section headings for key regions or tag existing visible headings as headers -- e.g., the main menu, the filter forms, the filtered search results...

not sure which pages are missing headings, but it seems at least the homepage might be.

**Comment by rlskoeser** [comment]
I couldn't find any pages besides the home page where this was a problem. I added a title there (the styles already were set to do the right thing).

**Comment by rlskoeser** [comment]
Home page has an h1 and no errors with WAVE browser test.
